.circleci/config.yml

```diff
@@ -245,12 +245,15 @@ jobs:
      - image: grafana/build-container:1.2.6
     working_directory: /go/src/github.com/grafana/grafana
     steps:
-      - checkout
```

Comment:
Interesting idea to store the whole project instead of checking it out from github. What is the thinking?

Reply:
since it's built and the code should not change between steps, persisting to workspace would be fast (and accurate) between steps of the build.  the paths are still giving me some trouble

Reply:
Does it work? I looked at it the other day and it looked like it took significantly longer to restore the workspace instead of checking out the code from github.

Reply:
What kind of problems are you having with the paths?

Reply:
it didn't work out, i went back to the checkout method